apiary_v2.apib

```diff
@@ -890,7 +908,31 @@ generating manufacturing files.
 
     + Body
 
-          {}
+          {
+            "reference": "YOUR ORDER REFERENCE",
+            "order_items": [
+              {
+                "design": "3cd42fe8-7491-11e8-adc0-fa7ae01bbebc",
+                "reference": "YOUR ORDER-ITEM 1 REFERENCE",
+                "quantity": 2,
+                "unit_price": 99.99,
+                "currency": "USD",
```

Comment:
Potentially here to do the `price` structure the same (either create or retrieve order and the same if we expose it for factory) `"unit_price" : {"value": 25, "currency": "USD"}`

Reply:
This is the payload they're sending. You want it the same there too?

Reply:
Make sense to be consistent maybe? On the way the sent and the way we serialise it here and factory. Even though I just realised that we're not exposing it now on factory which is fine.